micro-behaviors/cryptography/hashed-message-authentication-code.md

```diff
@@ -7,4 +7,4 @@
 
 Hashed Message Authentication Code
 ==================================
-Malware uses an HMAC schema.
\ No newline at end of file
+Malware uses an hashed-message-authentication-code schema.
```

Comment:
add acronym, change "an" to "a", and remove hyphens: "Malware uses a hashed message authentication code (HMAC) schema."